services/user.py

```diff
@@ -0,0 +1,37 @@
+from db.models import User
+
+
+def create_user(username: str, password: str,
+                email: str = None, first_name: str = None,
+                last_name: str = None) -> User:
+    user = User.objects.create_user(username=username, password=password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(pk=user_id)
```

Comment:
Ensure that the retrieval of a user by `user_id` handles the case where the user does not exist. You might want to catch the `User.DoesNotExist` exception to provide a more user-friendly error message or handle it appropriately.